authentik/stages/authenticator_email/apps.py

```diff
@@ -0,0 +1,11 @@
+"""Email"""
+
+from django.apps import AppConfig
+
+
+class AuthentikStageAuthenticatorEmailConfig(AppConfig):
+    """Email App config"""
+
+    name = "authentik.stages.authenticator_email"
+    label = "authentik_stages_authenticator_email"
+    verbose_name = "authentik Stages.Authenticator.Email"
```

Comment:
```suggestion
from authentik.blueprints.apps import ManagedAppConfig


class AuthentikStageAuthenticatorEmailConfig(ManagedAppConfig):
    """Email App config"""

    name = "authentik.stages.authenticator_email"
    label = "authentik_stages_authenticator_email"
    verbose_name = "authentik Stages.Authenticator.Email"
```

To automatically import and register the tasks in the worker